scss/_variables.scss

```diff
@@ -1545,6 +1545,7 @@ $accordion-border-color:                  $gray-500 !default; // Boosted mod
 $accordion-border-radius:                 $border-radius !default;
 $accordion-inner-border-radius:           subtract($accordion-border-radius, #{$accordion-border-width}) !default;
 
+$accordion-body-padding-top:              5px !default; // Boosted mod
```

Comment:
This is not a request for a modification but rather a question on how we tackle it more globally based on Bootstrap code. Having `$accordion-body-padding-y` and `$accordion-body-padding-top` seems not very logical/practical for end-users (see https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1449/files#r970460584)

Reply:
From what was said, we should split those variables if we add a new variable: `x-padding-y` -> `x-padding-top` + `x-padding-bottom`